sqlmesh/core/model/definition.py

```diff
@@ -38,7 +38,6 @@
     print_exception,
     serialize_env,
 )
-from sqlmesh.utils.pandas import filter_df_by_timelike
```

Comment:
can you delete this method now?

Reply:
seems like I can

Reply:
how do you filter pandas dataframes then?

Reply:
when it's translated to sql

Reply:
removed the function